proposals/0003_documentation-strategy.md

```diff
@@ -0,0 +1,130 @@
+
+A solution has been proposed by @tngTUDOR in the form of the Diataxis Framework. This framework was created and developed by Daniele Procida during his tenure at Divio. It formalizes different types of technical documentation according to the different needs of users. It is widely adopted in software development (e.g. at [Canonical, Cloudflare, Google, etc.](https://diataxis.fr/adoption/)).
+
+|    | serve our study | serve our work |
+| -: | :-------------: | :------------: |
+practical steps | __Tutorials__ <br/> _learning-oriented_ | __How-To Guides__ <br/> _task-oriented_ |
+theoretical knowledge | __Explanation__ <br/> _understanding-oriented_ | __Reference__ <br/> _information-oriented_ |
+
+Table 1: The [Diataxis framework](https://diataxis.fr/) 
+
+The framework makes an [important distinction based on the needs of the user](https://diataxis.fr/tutorials-how-to/): The purpose of a _tutorial_ is to provide a successful _learning experience_. The purpose of a _how-to guide_ is to to help the user _accomplish a task_.
+
+## Proposal
+
+All code added to the Brightway code base shall include docstrings conforming to the [Numpy Style Guide](https://numpydoc.readthedocs.io/en/latest/format.html). Existing code that does not conform to this standard shall be considered legacy code and shall be refactored. New code that does not conform to this standard shall not be merged into the Brightway code base.
+
+Based on the Diataxis framework, all Brightway documentation shall be structured as follows:
+
+|    | serve our study | serve our work |
+| -: | :-------------: | :------------: |
+practical steps | __Tutorials (=E-Book)__ <br/> Pages at <br/>  [`learn.brightway.dev`](https://github.com/brightway-lca/brightway-training) | __How-To Guides__ <br/> "Gallery" at <br/> [`docs.brightway.dev`](https://github.com/brightway-lca/brightway-documentation) |
+theoretical knowledge | __Explanation__ <br/> "LCA" at <br/> [`docs.brightway.dev`](https://github.com/brightway-lca/brightway-documentation)| __Reference__ <br/> "API Reference" at <br/>  [`docs.brightway.dev`](https://github.com/brightway-lca/brightway-documentation) |
+
+Table 2: The [Brightway](https://github.com/brightway-lca) implementation of the [Diataxis framework](https://diataxis.fr/)
+
+##### Reference
+
+This shall contain the API reference for the various Brightway modules (`bw_agg`, `bw_calc`, `bw_data`, ...). This is compiled automatically from the function docstrings by the [Sphinx package](https://www.sphinx-doc.org/en/master/).
```

Comment:
blocking: let's rephrase this to say that the reference shall contain the API reference for the "core brightway modules". They are defined elsewhere as such.